line/index.js

```diff
@@ -228,6 +264,15 @@ parseData.load(dataFile, { dateFormat, yMin, joinPoints, highlightNames })
           .attr('id', d => d.name)
           .call(myChart);
 
+        currentFrame.plot()
+          .selectAll('lines')
```

Comment:
Should be `.lines`.

Reply:
Good spot, explains why I can't get the save png to recognise CCS I hope